### ISSUE TITLE
Default images for `helm server install` need to be updated

### ISSUE THREAD

**Issue by jackgr** [kubernetes/helm#538]
Currently, the defaults are set to the images for DM v1.2.1. At this point, it feels like people will want to use latest. Certainly needs to be done by MVP:
```
HELM_MANAGER_IMAGE=gcr.io/dm-k8s-prod/manager:latest
HELM_EXPANDYBIRD_IMAGE=gcr.io/dm-k8s-prod/expandybird:latest
HELM_RESOURCIFIER_IMAGE=gcr.io/dm-k8s-prod/resourcifier:latest
```

**Comment by michelleN** [comment]
When we do an MVP release, we'll want to use semantically versioned tags to refer to images instead of latest or git tags.

cc/ @adamreese and @runseb

**Comment by jackgr** [comment]
Agreed.

**Comment by jackgr** [comment]
#557

**Comment by jackgr** [comment]
Fixed by #556.
